extensions/sql-migration/src/models/migrationLocalStorage.ts

```diff
@@ -62,6 +66,20 @@ export class MigrationLocalStorage {
 		return result;
 	}
 
+	public static async refreshMigrationAzureAccount(migration: MigrationContext): Promise<void> {
+		if (migration.azureAccount.isStale) {
```

Comment:
Should we also open the azure accounts dialog when we cannot refresh the credentials?

Reply:
that would help show issue could not be fixed, but wonder if it would become problematic when the account is disabled or lacking subscriptions when causing the user to be prompted several times.  Perhaps a property can be added to the migration context that indicates the data is stale to help guide the user to refresh the account.